Formula/maturin.rb

```diff
@@ -0,0 +1,21 @@
+class Maturin < Formula
+  desc "Build and publish Rust crates as Python packages"
+  homepage "https://github.com/PyO3/maturin"
+  url "https://github.com/PyO3/maturin/archive/refs/tags/v0.10.6.tar.gz"
+  sha256 "e6a9a67cc62ffe248654e60e7ec211bf23319c4c936ad87022f7a1fd0997430d"
+  license "MIT"
+  head "https://github.com/PyO3/maturin.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "Build and publish crates with pyo3", shell_output("#{bin}/maturin --help 2>&1")
+
+    system "cargo", "new", "hello_world", "--bin"
+    system "#{bin}/maturin", "build", "-m", "hello_world/Cargo.toml", "-b", "bin", "-o", "dist"
```

Comment:
This is better. Can we test that this created a Python package correctly? We can probably use `assert_predicate` with `:exist?` and `assert_match` to read substrings from any generated files.

Reply:
Not sure how to use `assert_match` to read a `.whl` file. (A zip file)

Reply:
Update to test package installation with python3.

Reply:
The new test looks good. Thanks for your work on this.